casa_formats_io/tests/test_casa_low_level_io.py

```diff
@@ -275,3 +276,35 @@ def test_vector_columns(tmp_path):
     assert_equal(t['short'], t2['short'])
     assert_equal(t['int'], t2['int'])
     assert_equal(t['double'], t2['double'])
+
+
+# Test reading tables in the casadata package
+
+try:
+    from casadata import datapath
+except ImportError:
+    CASADATA_TABLES = []
+    CASADATA_INSTALLED = True
+else:
+    # TODO: In future we can include more of the tables in the testing
+    CASADATA_TABLES = [os.path.dirname(x) for x in glob.glob(os.path.join(datapath, 'geodetic', 'IERS*', 'table.dat'), recursive=True)]
```

Comment:
@keflavich - a real stress test is to actually glob all the tables in the casadata package - I tried that and got some failures (and a lot of passes too!) so I think I'll have to tackle it progressively but it's nice to have such an extensive set of example tables!